### PR TITLE
Private DS

### PR DESCRIPTION
**Pull request by amitzur**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const wrapeAndExportPlugin = require('./src/wrapeAndExportPlugin');
 const prepareForClientFunctionPlugin = require('./src/prepareForClientFunctionPlugin');
 
 module.exports = {
-  input: 'node_modules/@applitools/dom-snapshot/src/browser/processPageAndSerialize.js',
+  input: 'node_modules/@applitools/dom-snapshot/dist/processPageAndSerializeCjs.js',
```
Comment (amitzur) [inline rollup.config.js:11]: `/dist/processPageCjs` along with `serializeResources`

File: tests/e2e/testcafe-tests/selector.testcafe.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /* global fixture, test */
 import {Selector} from 'testcafe';
 import path from 'path';
-import Eyes from '../../src/eyes';
-const configPath = path.resolve(__dirname, '../fixtures/applitools.config.js');
+import Eyes from '../../../';
+const configPath = path.resolve(__dirname, '../../fixtures/applitools.config.js');
 const eyes = new Eyes({configPath});
 
 fixture.only`Selector`.page`http://localhost:7272/helloworld.html`
```
Comment (tourdedave) [inline tests/e2e/testcafe-tests/selector.testcafe.js:8]: @amitzur Did you ever notice this? This is the only e2e test that runs. Been that way for 5 months...

Comment (amitzur) [reply tests/e2e/testcafe-tests/selector.testcafe.js:8]: phhhh, i might have and I don't remember.

File: tests/it/dom-snapshot.ie11.it.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  it('works', async () => {
+    let testcafe = null;
+    let failedCount;
+    await createTestCafe('localhost', 1337)
+      .then(tc => {
+        testcafe = tc;
+        const runner = testcafe.createRunner();
+        return runner
+          .src([testPath])
+          .browsers(['saucelabs:Internet Explorer@11.0:Windows 7'])
```
Comment (tourdedave) [inline tests/it/dom-snapshot.ie11.it.test.js:30]: Throws `Error: 'assign' is not a function`

I'm assuming the latest version of the SDK gets a similar result on IE11.

File: tests/it/dom-snapshot.edge.it.test.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  it('works', async () => {
+    let testcafe = null;
+    let failedCount;
+    await createTestCafe('localhost', 1337)
+      .then(tc => {
+        testcafe = tc;
+        const runner = testcafe.createRunner();
+        return runner
+          .src([testPath])
+          .browsers(['saucelabs:MicrosoftEdge@18.17763:Windows 10'])
```
Comment (tourdedave) [inline tests/it/dom-snapshot.edge.it.test.js:30]: The test works when run by itself, but not as part of a suite. Seems like something odd in the `sauce-connect-launcher`

File: rollup.config.js (hunk 1 above)
Comment (tourdedave) [reply rollup.config.js:11]: FYI - the config as-is in the PR works.

File: tests/it/dom-snapshot.ie11.it.test.js (hunk 3 above)
Comment (tourdedave) [reply tests/it/dom-snapshot.ie11.it.test.js:30]: But `assign` isn't support in IE11. Seems like a case that's not transpiled by TestCafe. I'll check.

Comment (tourdedave) [reply tests/it/dom-snapshot.ie11.it.test.js:30]: Confirmed the same behavior on `master`

**Pull request merged**